File: objective-c/two-fer/TwoFer.m
#import "TwoFer.h"

@implementation TwoFer

+ (NSString *) twoFerWithName:(NSString * _Nullable)name {
    NSString * nameToInsert = (name == nil) ? @"you" : name;

    return [NSString stringWithFormat:@"One for %@, one for me.", nameToInsert];
}

@end

File: swift/two-fer/Sources/TwoFer/TwoFer.swift
//Solution goes in Sources

struct TwoFer {

    static func twoFer(name: String? = nil) -> String {
    	guard let string = name else {
    		return "One for you, one for me."
    	}

    	return "One for \(string), one for me."
    }

}
